ntbs-service/Pages/Notifications/Edit/_SaveButton.cshtml

```diff
@@ -1,21 +1,39 @@
+@using ntbs_service.Helpers
 @using ntbs_service.Models.Enums
 @model ntbs_service.Models.Entities.Notification
 
 @{
+    var isDraft = Model.NotificationStatus == NotificationStatus.Draft;
+    var isNonEditPage = (bool?) ViewData["IsNonEditPage"] ?? false;
+    var currentPage = (string) ViewData["CurrentPage"];
+
     var buttonName = "";
-    if (@ViewData.ContainsKey("ShowContinueButton") && (bool)@ViewData["ShowContinueButton"])
+    if (isNonEditPage)
     {
         buttonName = "Continue";
-    } else {
-        buttonName = Model?.NotificationStatus == NotificationStatus.Draft ? "Save and continue" : "Save";
     }
-    var id = @ViewData.ContainsKey("id") ? @ViewData["id"] : "save-button";
+    else
+    {
+        buttonName = isDraft ? "Save and continue" : "Save";
+    }
+
+    var id = ViewData.ContainsKey("id") ? ViewData["id"] : "save-button";
 }
-<button
-    nhs-button-type="Standard"
-    classes="ntbsuk-button--primary"
-    name="actionName"
-    value="Save"
-    id=@id >
-    @buttonName
-</button>
\ No newline at end of file
+
+@* Conditional excluding (!isDraft && isNonEdit) *@
```

Comment:
Not sure how much this helps with explaining the behaviour, as it just gives a mathematical transformation of the condition. I'd recommend we either cure this comment out completely, or we explain the context of *why* we're not showing the button for a that condition